pom.xml

```diff
@@ -41,6 +41,10 @@
                     <groupId>org.apache.tinkerpop</groupId>
                     <artifactId>gremlin-core</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
Excluding commons-text here as this version of janusgraph is using a very old version which is creating conflicts. I've done some testing and the janusgraph serialization is still functioning correctly with a newer version of this dependency.